### PR TITLE
add sms send api endpoint

### PR DESCRIPTION
**Pull request by galvani**
Describes the new endpoint and adds sendSMS function to the smsApi

### CODE REVIEW

**Comment by escopecz** [comment]
For the record the new endpoint is implemented here: https://github.com/mautic/mautic/pull/5854

**Comment by galvani** [comment]
@escopecz tests are added. would you?

File: tests/Api/SmsesTest.php [hunk 1]
```diff
@@ -51,4 +62,50 @@ public function testBatchEndpoints()
...
+            'name'      => 'API Test Message',
+            'message'   => 'You should not get this',
+        ]);
+
+        if ($contact && $message) {
+            $requestResponse =  $this->api->makeRequest($this->endpoint.'/' . $message['id'] . '/contact/' . $contact['id'] . '/send');
+            $this->assertCount(1, $requestResponse['errors']);
+            $response = $this->api->sendSMS($message['id'], $contact['id']);
+
+            $this->assertTrue(!$response['result']['sent']);
```
Comment (escopecz) [inline tests/Api/SmsesTest.php]: Is this test expecting that the sent will fail?

Comment (galvani) [reply tests/Api/SmsesTest.php]: yes, it is. actually it fails for different reason than I intended but I am just to test that Twillio transport will be called. There are other tests to test the transportChain and transport calls itself.

**Pull request closed without merging**